test/index.test.js

```diff
@@ -18,12 +18,19 @@ function resultTester(result) {
 }
 
 test.serial(`Should process a playlist URL`, async t => {
-  const url = 'https://iptv-org.github.io/iptv/categories/classic.m3u'
+  const url = 'https://iptv-org.github.io/iptv/categories/auto.m3u'
   const results = await iptvChecker(url, { timeout: 2e3, parallel: 1 })
 
   t.true(resultTester(results))
 })
 
+test.serial(`Should process a playlist absolute path`, async t => {
```

Comment:
The provided `test/input/dummy.m3u` is not an absolute path, but relative.

The test below this one already tests an absolute file path, defined as constant `playlistPath`

Reply:
Sorry, I missed it.